sdks/v4-sdk/src/PositionManager.ts

```diff
@@ -114,21 +114,19 @@ export interface CollectSpecificOptions {
   recipient: string
 }
 
-export interface TransferOptions {
+export interface MigrateOptions {
   /**
-   * The account sending the NFT.
+   * Whether the mint is part of a migration from V3 to V4.
    */
-  sender: string
-
+  migrate?: boolean
   /**
-   * The account that should receive the NFT.
+   * The additional currency that needs to be transferred if migrating from out of range to in range or out of range to opposite side out of range
    */
-  recipient: string
-
+  neededCurrency?: Currency
   /**
-   * The id of the token being sent.
+   * The amount of additional currency that needs to be transferred if migrating from out of range to in range or out of range to opposite side out of range
    */
-  tokenId: BigintIsh
+  neededAmount?: BigintIsh
```

Comment:
Should `neededAmount` and `neededCurrency` be combined into a single `CurrencyAmount` type?
You should never have one without the other, so i think a single optional `CurrencyAmount` would make more sense